File: Beecrowd/1156 Sequência S II/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        double aux, s = 1;
        double aux2 = 1;

        for (int i = 3; i < 40; i++) {
            if (i % 2 != 0) {
                aux2 *= 2;
                aux = i / aux2;
                s += aux;
            }
        }
        System.out.printf("%.2f%n", s);

        sc.close();
    }
}

File: Beecrowd/1156 Sequência S II/main.cpp
#include <iostream>
 
using namespace std;

int main(){
    long double aux, S = 1;
    double aux2 = 1;
    int i;
    
    for(i = 3; i < 40; i++){
      if(i % 2 != 0){
        aux2 *= 2;
        aux = i / aux2;
        S += aux;
      }
    }
    cout << fixed;
    cout.precision(2);
    cout << S << endl;
    
	return 0;
}